src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -0,0 +1,119 @@
+import math
+import os
+from functools import partial
+from typing import Any, Mapping, Optional, Sequence, Tuple, Union
+
+import tiledb
+from tiledb.cloud.utils import logger
+
+
+def ingest(source: Union[Sequence[str], str],
+           output: str, 
+           access_key_id: str, 
```

Comment:
Lets support roles here too, perhaps we should generically take a dictionary of configs to pass to tiledb's vcf?

Reply:
Yeap a `config` dict would be better.